float64 theta3
geometry_msgs/Pose pose
---
ik[] ikSolutions 
bool status

